files/es/web/api/battery_status_api/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: API de Estado de Bateria
+slug: Web/API/Battery_Status_API
+tags:
+  - API
+  - Apps
+  - Firefox OS
+  - Mobile
+translation_of: Web/API/Battery_Status_API
+original_slug: WebAPI/Estado_de_Bateria
```

Comment:
vi estó en la doc en Inglés, debemos agregarlo?
```suggestion
original_slug: WebAPI/Estado_de_Bateria
browser-compat: api.BatteryManager
```

Reply:
No se a que se refiere exactamente, pero creo que lo agregamos por si acaso.

Reply:
If I may interject (still in English for now, unfortunately), this `browser-compat` line should be present in the front-matter of a file when using the `{{Compat}}` macro (it's not *strictly* necessary but it's good practice). This method is **preferred** to writing it as `{{Compat("api.BatteryManager")}}`.